__tests__/boot.test.ts

```diff
@@ -1,85 +1,140 @@
-import io = require('@actions/io')
-import os = require('os')
-import fs = require('fs')
-import path = require('path')
+import * as _core from '@actions/core'
+import * as _exec from '@actions/exec'
+import * as _io from '@actions/io'
+import * as _ioUtil from '@actions/io/lib/io-util'
```

Comment:
Note on `io-util` usage: it's hard to just mock `fs` module, since it crashes `@actions/io` module ([here](https://github.com/actions/toolkit/blob/main/packages/io/src/io-util.ts#L16)). However, it seems to be a good idea to replace `fs` usage to `@actions/io/lib/io-util`, as it gives simplicity of synchronous functions while remaining asynchronous via promises usage.

Reply:
UPD: instead of poorly documented `@actions/io/lib/io-util` I've created special `fs.ts` file that re-exports functions that are used in the action.